src/bridge/secrets/ocw_studio/ocw_studio.ci.yaml

```diff
@@ -0,0 +1,77 @@
+---
+github_shared_secret: ENC[AES256_GCM,data:Pdrn+lFOh9Rb53XpxIxzZvLpzNKIe6lbGXvT1Gbu3ZsnaKseUjqKHcqCA5l3RdHJahSBOWzvgGn5BWG/5El4r8GQ,iv:ug0p3Kgqc/p2+04rO8hZTjlE3MnbXS+/Awl/n1GFy14=,tag:NYOrJH9meTTCE9MamXHcLQ==,type:str]
```

Comment:
The data in Vault also contains a `github_app_private_key` which we either need to include in these SOPS files or determine with the devs whether it is still needed.